python/scripts/manifest.csv

```diff
@@ -31,6 +31,8 @@ use,census_margin_of_error,acs5_2012_moe,api,7/1/2013,utf-8,../../../data,https:
 use,census_margin_of_error,acs5_2013_moe,api,7/1/2013,utf-8,../../../data,https://s3.amazonaws.com/housinginsights/,raw/apis/20170606/acs5_2013_moe.csv,
 use,census_margin_of_error,acs5_2014_moe,api,7/1/2013,utf-8,../../../data,https://s3.amazonaws.com/housinginsights/,raw/apis/20170606/acs5_2014_moe.csv,
 use,census_margin_of_error,acs5_2015_moe,api,7/1/2013,utf-8,../../../data,https://s3.amazonaws.com/housinginsights/,raw/apis/20170606/acs5_2015_moe.csv,
+use,census_tract_to_neighborhood_cluster,tract2010_cluster2000,api,7/1/2013,utf-8,../../../data,https://s3.amazonaws.com/housinginsights/,raw/geographic_data/Tract_weights/Wt_tr10_cltr00.csv,

```

Comment:
These two files should actually be denoted as 'manual' not as 'api' . This designation is not yet used but will be how we allow code to update the manifest any time we run the get_api_data.py file automatically on the server.